src/react-intl-messages.ts

```diff
@@ -972,7 +972,7 @@ export default {
         [FM.TOOLTIP_ACTION_RESPONSE_ROW3]: '"Hi[, $name]"',
 
         [FM.TOOLTIP_ACTION_REQUIRED]: 'Action will not be selected unless Memory contains values for these Entities',
-        [FM.TOOLTIP_ACTION_REQUIRED_TITLE]: 'Required Entities',
+        [FM.TOOLTIP_ACTION_REQUIRED_TITLE]: 'Required Entities or Conditions',
```

Comment:
I had thought that we could drop `Entities` wording here. The entity being present is just another condition.

Instead of user having to think entities and conditions are different, the entity being present is just another condition where the entity value can be anything as opposed to conditions where entity value must be an Enum value or must be greater than some number.  Would simply concepts for users and avoid exposure to ML concepts like 'entity' average user likely more commonly understands the context and implications of word "condition" that what it means to require an entity.

Reply:
Interesting idea.  Run it by Swadheen?

Reply:
@swadheen Thoughts?

Reply:
Pinging @swadheen again..,

Reply:
Sounds good.. Thanks!